themes/jquery/css/base.css

```diff
@@ -676,14 +676,30 @@ table {
 }
 
 #content th {
-	font-family: "klavika-web", "Helvetica Neue", Helvetica, Arial, Geneva, sans-serif;
+	font-family: Cairo, "Helvetica Neue", Helvetica, Arial, Geneva, sans-serif;
 }
 
 #content th,
 #content td {
 	padding: .5em;
 }
 
+/* ==========================================================================
+   Cairo Font
+   ========================================================================== */
+
+@font-face {
+	font-family: Cairo;
+	src: url(fonts/Cairo/Cairo-Regular.ttf) format("truetype");
```

Comment:
Is there no WOFF/WOFF2 version?

Reply:
No, but from your other comment, it sounds like you prefer to remove woff everywhere.

Reply:
Nvm, I misread. You wanted to just remove eot. Do we need woff, though?

Reply:
WOFF is the modern format supported everywhere; I think even WOFF2 works everywhere now. We should only ever need WOFF (and WOFF2, if available), other versions can be dropped.

Reply:
See, I think we should only ever need open types like TTF and OTF. WOFF is not needed. Besides, Google doesn't have WOFF downloads. I can leave WOFF for fontawesome if you prefer, though.

Reply:
Ok, I hadn't actually heard of WOFF. It seems it's just web-only TTF. We can convert it, but I'd have to dig into the Google Font license to see if that's allowed.

Reply:
Cairo uses https://openfontlicense.org/, so it *should* be okay to convert. Can we remove TTF everywhere then?

Reply:
I have confirmed that woff loads in IE9